_processSteps.py

```diff
@@ -473,3 +481,30 @@ def run_ingest_insarmaps(inps):
 
     if inps.stopinsarmaps:
         logger.log(loglevel.DEBUG, 'Exit as planned after insarmaps')
+
+
+###############################################################################
+
+
+def overwrite_stored_date(dset):
```

Comment:
Is this function correct? I don't see how the updated lines are ever written to the file.

Something like [this](https://www.google.com/search?q=overwrite+specific+line+in+file+python&oq=overwrite+specific+line+in+file+python&aqs=chrome..69i57j0l3.9547j1j7&sourceid=chrome&ie=UTF-8) seems a bit cleaner, but if there are multiple versions of this same program running, it could lead to concurrency issues.

Reply:
Hmm. Thats odd. I thought I removed that function. You are correct in that it doesn't work properly (ie. not writing new data). I was messing around with whether it was better to include at the end of `process_rsmas` itself or let `run_operations` handle it. I must have missed remove the function from the final push.